docs/getting_started/dask.rst

```diff
@@ -48,9 +48,13 @@ supported, it still works by falling back to running pandas. One of the key feat
 of being a drop-in replacement is that not only will it work for existing code, if a
 user wishes to go back to running pandas directly, they are not locked in to using Modin and can switch between Modin and pandas at no cost. In other words, Modin notebooks can be converted to and from pandas as the user desires.
 
+.. figure:: ../img/api_coverage_comparison.png
```

Comment:
Can we have the pictures in `.svg` (both as source and used to display)? that way they're editable in the future (and it's how we started doing in the new documentation PRs mostly).

Reply:
Got it! I changed the image to `.svg` and made sure that the text is editable.